README: What to Tweet (#11)
probot/twitter: README.md

Follow [these instructions](./tweets) for a detailed guide, but the short version is: create a `.tweet` file in `/tweets` and open a pull request. We'll review it, and once it's merged, the [@ProbotTheRobot](https://twitter.com/ProbotTheRobot) Twitter account will tweet it out!

## What to Tweet

We encourage anyone to craft and form a tweet, by opening a PR in the tweets folder. Possible topics include:

> - Interesting usage / us cases of Probot

Reviewer: ```suggestion
- Interesting usage/use-cases of Probot
```

> - Tutorials / projects GitHub workflow automation

Reviewer: ```suggestion
- Tutorials/projects about GitHub workflow automation
```

- News about Probot and related projects
- Other items relevant to the Probot community

> If you build an app with Probot, we are happy to share the word! The app should either be open soure or should mention that it’s built with Probot in the marketplace / app description as well as the website if one exists.

Reviewer: ```suggestion
If you build an app with Probot, we are happy to share the word! The app should either be open source or should mention that it’s built with Probot, in the marketplace/app description as well as the website if one exists.
```

## Accepting a tweet

> One accept by a maintainers is enough for a merge.

Reviewer: ```suggestion
One acceptance by a maintainer is enough for a merge.
```
